runtime/reconcilers/model.go

```diff
@@ -1009,6 +1021,7 @@ func (r *ModelReconciler) executeAll(ctx context.Context, self *runtimev1.Resour
 		// Start the worker goroutines
 		grp, ctx := errgroup.WithContext(ctx)
 		counter := &atomic.Int64{}
+		duration := &atomic.Int64{}
```

Comment:
Consider just making `totalExecDuration` an `atomic.Int64` from the start, then you don't need two different but similarly named variables